src/solvers/flattening/bv_conversion_exceptions.h

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************\
+
+Module: Bit vector conversion
+
+Author: Diffblue Ltd.
+
+\*******************************************************************/
+
+/// \file
+/// Exceptions that can be raised in bv_conversion
+
+#ifndef CPROVER_SOLVERS_FLATTENING_BV_CONVERSION_EXCEPTIONS_H
+#define CPROVER_SOLVERS_FLATTENING_BV_CONVERSION_EXCEPTIONS_H
+
+#include "flatten_byte_extract_exceptions.h"
+#include "flatten_byte_operators.h"
+#include <util/endianness_map.h>
+#include <util/arith_tools.h>
```

Comment:
Nit picking: can the includes be grouped by directory and be lexicographically sorted?